homeassistant/components/notify/nfandroidtv.py

```diff
@@ -110,19 +116,26 @@ def __init__(self, remoteip, duration, position, transparency, color,
         self._default_color = color
         self._default_interrupt = interrupt
         self._timeout = timeout
-        self._icon_file = os.path.join(
-            os.path.dirname(__file__), '..', 'frontend', 'www_static', 'icons',
-            'favicon-192x192.png')
+        self._icon_file = None
+        if icon:
+            self._icon_file = icon
+        else:
+            try:
+                import hass_frontend
+                default_icon = os.path.join(
+                    hass_frontend.__path__[0], 'icons', 'favicon-192x192.png')
+                if os.path.exists(default_icon):
+                    self._icon_file = default_icon
+            except ImportError:
```

Comment:
Actually, it's weird that this component even depends on the frontend at all. I wonder if we should include a Home Assistant logo in this repo for such circumstances.

Reply:
In most situations an icon probably is not required. It's rather a problematic design choice at the TV app. But since I can't influence that there's no other way. Before the changes to the frontend the icons where here, so it wasn't an issue. Having an icon here would be fine for me. Any suggestions where it should go?

Reply:
Why don't we start with just the bytes of an empty gif, you could embed that in the code.